feconf.py

```diff
@@ -101,6 +101,9 @@
     TRAINING_JOB_STATUS_FAILED: [TRAINING_JOB_STATUS_NEW]
 }
 
+# The maximum number of activities allowed in the playlist of the learner.
```

Comment:
Append to comment: This limit applies to both the explorations playlist and the collections playlist.